test/common/http/http2/BUILD

```diff
@@ -41,16 +41,17 @@ envoy_cc_test(
     name = "codec_impl_test",
     srcs = ["codec_impl_test.cc"],
     shard_count = 5,
+    # The default codec is the legacy codec. Override runtime flag for testing new codec.
+    args = [
+        "--runtime-feature-override-for-tests=envoy.reloadable_features.new_codec_behavior",
+    ],
     tags = ["fails_on_windows"],
     deps = CODEC_TEST_DEPS,
 )
 
 envoy_cc_test(
     name = "codec_impl_legacy_test",
     srcs = ["codec_impl_test.cc"],
```

Comment:
For completeness, can you add runtime-feature-disable-for-tests to the legacy test?

Reply:
all set! nice. it'll be easier to flip them later too.